bin/build.js

```diff
@@ -1,29 +1,36 @@
-/* eslint-disable no-console, no-sync */
+/* eslint-disable no-console, no-process-exit */
+const chalk = require('chalk');
+const emoji = require('node-emoji');
 const path = require('path');
-const fs = require('fs');
 
-const SVG_FILENAME = 'core-icons.js';
-const SVG_FILENAME_MIN = 'core-icons.min.js';
-const SRC_PATH = path.join(__dirname, '../src');
-const DIST_PATH = path.join(__dirname, '../dist');
+const concatSvg = require('./concat-svg');
+const parseArgs = require('minimist');
 
-const svgToSymbol = (file, xml) => String(xml) // Work with file as string
-  .replace('>', ` id="${file.slice(0, -4)}">`) // Add filename as id
-  .replace(/(<\/?)svg/gi, '$1symbol')          // Convert to symbols
-  .replace(/\s*xmlns=[^\s>]+/gi, '')           // Remove xmlns
-  .replace(/\s*([<>])\s*/g, '$1');             // Strip white space around tokens
+const defaultOptions = {
+  svgFileName: 'core-icons.js',
+  svgFileNameMin: 'core-icons.min.js',
+  srcPath: path.join(__dirname, '../src'),
+  distPath: path.join(__dirname, '../dist'),
+  banner: `Copyright (c) 2015-${new Date().getFullYear()} NRK <opensource@nrk.no> `
```

Comment:
banner is a thing other build systems use I think, that's why I used it. Unsure if it's actully a term.